src/CommunityVoices/App/Website/Config/Routes.json

```diff
@@ -569,11 +569,28 @@
         }
     },
     "getAllErrors": {
-        "notation": "/errors",
+        "notation": "/errors-log",
         "method": "get",
         "defaults": {
             "resource": "DisplayError",
             "action": "getAllErrors"
         }
+    },
+    "getSomeErrors": {
+        "notation": "/errors-log/{lines}",
+        "method": "get",
+        "defaults": {
+            "resource": "DisplayError",
+            "action": "getSomeErrors"
+        }
+    },
+    "postErrorsSearch": {
+        "notation": "/errors-log/search",
```

Comment:
I thought we had talked about transitioning this into a query string?

Reply:
Meant to get rid of